src/translators/engines/deepl.ts

```
@@ -122,8 +140,237 @@ class DeepL extends TranslateEngine {
+   * 1. Deletes all existing glossaries for the given source-/target language pair
+   * 2. Creates a new glossary
+   *
+   * @param targetLanguage
+   * @param sourceLanguage
+   * @param content CSV-formatted glossary
+   */
+  public async updateGlossary(targetLanguage: string, sourceLanguage: string, content: string) {
+    try {
+      const glossaries = await this.getGlossaryIds(targetLanguage, sourceLanguage)
+
+      glossaries.forEach(async id => {
```

ESLint

[error] Expected parentheses around arrow function argument having a body with curly braces.

(arrow-parens)